dist/package.json

```diff
@@ -11,6 +11,9 @@
   },
   "keywords": [
     "angular2",
+    "angular4",
+    "angular5",
+    "angular4",
```

Comment:
There is 2 keywords `angular4`. Is this supposed to be like this?

Reply:
hah no... well spotted